<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-NKG6J0Q" ignored="0" name="Test3" tests="2" failures="1" timestamp="22 Jan 2021 06:28:39 GMT" time="46.763" errors="0">
  <testcase name="addempleelistlabletest" time="12.275" classname="com.TestCases.EmployeepagelistTest"/>
  <testcase name="empleelistTest" time="20.847" classname="com.TestCases.EmployeepagelistTest">
    <failure type="java.lang.IllegalArgumentException" message="The workbook already contains a sheet of this name">
      <![CDATA[java.lang.IllegalArgumentException: The workbook already contains a sheet of this name
at org.apache.poi.xssf.usermodel.XSSFWorkbook.createSheet(XSSFWorkbook.java:650)
at com.Page.Employeelistpage.getelistOfEmployees(Employeelistpage.java:46)
at com.TestCases.EmployeepagelistTest.empleelistTest(EmployeepagelistTest.java:52)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- empleelistTest -->
</testsuite> <!-- Test3 -->
